platform/broadcom/sonic-platform-modules-dell/s5296f/scripts/s5296f_platform.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
```

Comment:
Indents in this file are very inconsistent. Please replace tabs with 4 spaces, and make all levels of indentation 4 spaces.

Reply:
Done